dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.javaee.websocket-1.0.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.javaee.websocket-1.0
+WLP-DisableAllFeatures-OnConflict: false
+singleton=true
+-features=com.ibm.websphere.appserver.eeCompatible-7.0; ibm.tolerates:="6.0, 8.0"
+-bundles=com.ibm.websphere.javaee.websocket.1.0; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.websocket:javax.websocket-api:1.0"
+kind=ga
+edition=core
```

Comment:
Why does this one not have `WLP-Activation-Type: parallel` ?

Reply:
This was based off the public feature where the 1.1 included the header but 1.0 didn't. Let me investigate if there was a reason behind this. New features should include the header so we would potentially want to include it here as well if there isn't any conflict with the bundles

Reply:
Created issue #17894 to verify adding the header to the feature